libs/dh/core/routing/project.json

```
@@ -0,0 +1,8 @@
+{
+  "name": "dh-shared-routing",
+  "$schema": "../../../../node_modules/nx/schemas/project-schema.json",
+  "projectType": "library",
+  "sourceRoot": "libs/dh/shared/routing/src",
```

suggestion: the path should match the actual library path 

```suggestion
  "sourceRoot": "libs/dh/core/routing/src",
```